Patches/RWY Dragon's Descent Void Dwellers/Races_Animal_SpaceDragon_Base.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<!-- I took 3 months off of RimWorld and I've forgotten how to do patches. Great. -->
```

Comment:
😢

Reply:
That's not me, 😅 it's aluminium alman which made the original dragon's descent patch, I never take breaks.